<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Paper Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Search Keyword</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Author:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_author">
                     <property name="placeholderText">
                      <string> Enter the keyword of author</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Title:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEdit_keyword">
                     <property name="placeholderText">
                      <string> Enter the keyword of title</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="toolTip">
                    <string/>
                   </property>
                   <property name="text">
                    <string>Event Type: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_eventType">
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::DefaultContextMenu</enum>
                   </property>
                   <property name="toolTip">
                    <string extracomment="Tip: You must select an event type to search."/>
                   </property>
                   <property name="toolTipDuration">
                    <number>30</number>
                   </property>
                   <property name="inputMethodHints">
                    <set>Qt::ImhNone</set>
                   </property>
                   <property name="currentText">
                    <string/>
                   </property>
                   <property name="currentIndex">
                    <number>-1</number>
                   </property>
                   <item>
                    <property name="text">
                     <string>Poster</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Oral</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Spotlight</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Poster, Oral, Spotlight</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="pushButton_search">
                   <property name="text">
                    <string>Search</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_clearTable">
                   <property name="text">
                    <string>Clear Table</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_clearQuery">
                   <property name="text">
                    <string>Clear Query</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Total: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_total">
                  <property name="cursor">
                   <cursorShape>IBeamCursor</cursorShape>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Page:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_page"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableView" name="tableView">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="pushButton_first">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>icon/first.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_previous">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>icon/previous.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_next">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>icon/next.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_last">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>icon/last.png</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1,10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Authors</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextBrowser" name="textBrowser_authors"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Title</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextBrowser" name="textBrowser_title"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Abstract</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextBrowser" name="textBrowser_abstract"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="pushButton_goTab2">
                <property name="text">
                 <string>Go to &quot;Full Content&quot;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_image_Tab1">
                <property name="text">
                 <string>Go to &quot;Paper Image&quot;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_scholar">
                <property name="text">
                 <string>Google Scholar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QPushButton" name="pushButton_save">
                <property name="text">
                 <string>Save to Excel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_exit">
                <property name="text">
                 <string>Exit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Andale Mono&quot;;</string>
          </property>
          <property name="text">
           <string>Copyright © 2023 by Yvette</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Full Content</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Full Content</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser_fullContent"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0,0,0">
            <item>
             <widget class="QPushButton" name="pushButton_backTab1">
              <property name="text">
               <string>Go to &quot;Paper Search&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_image_Tab2">
              <property name="text">
               <string>Go to &quot;Paper Image&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_scholar_Tab2">
              <property name="text">
               <string>Google Scholar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_exit_Tab2">
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Andale Mono&quot;;</string>
          </property>
          <property name="text">
           <string>Copyright © 2023 by Yvette</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Paper Image</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11" stretch="5,1">
          <item>
           <widget class="QLabel" name="label_image">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,1,1">
            <item>
             <widget class="QPushButton" name="pushButton_paperSearch_Tab3">
              <property name="text">
               <string>Go to &quot;Paper Search&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_FullContent_Tab3">
              <property name="text">
               <string>Go to &quot;Full Content&quot;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_exit_Tab3">
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
